### ISSUE TITLE
[Bug]: Pymilvus 2.2.4 depends on grpcio 1.47.X doesn't install on Apple Silicon

### ISSUE THREAD

**Issue by blasscoc** [milvus-io/pymilvus#1347]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

If I pip install pymilvus it resolves the older grpcio versions:

Collecting pymilvus
  Using cached pymilvus-2.2.4-py3-none-any.whl (128 kB)
Collecting grpcio<=1.48.0,>=1.47.0
  Using cached grpcio-1.47.5.tar.gz (21.9 MB)
  Preparing metadata (setup.py) ... done
Collecting grpcio-tools<=1.48.0,>=1.47.0
  Using cached grpcio-tools-1.47.5.tar.gz (2.2 MB)
  Preparing metadata (setup.py) ... done

### Expected Behavior

According to your requirements.txt, Pymilvus depends on 1.53.0. So I suppose the expected behaviour is that it would install that version.

### Steps/Code To Reproduce behavior

```markdown
pip install pymilvus
```

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
Apple M1

- Method of installation (Docker, or from source):
pip
- Milvus version (v0.3.1, or v0.4.0):
what every you have on pip
- Milvus configuration (Settings you made in `server_config.yaml`):
I don't actually use your stuff, it's just screwing up a dependency on a project I do use.
```

**Comment by XuanYang-cn** [comment]
Dup with #1308 

Shouldn't be a problem after pymilvus==2.2.5, we're going to release 2.2.5 this Friday.

**Comment by XuanYang-cn** [comment]
available on pymilvus==2.2.5
